src/relax/op/tensor/manipulate.cc

```diff
@@ -790,5 +790,125 @@ TVM_REGISTER_OP("relax.broadcast_to")
     .add_argument("shape", "Shape", "The target shape.")
     .set_attr<FInferStructInfo>("FInferStructInfo", InferStructInfoBroadcastTo);
 
+void CheckCollapseShape(const Call& call, const BlockBuilder& ctx,
+                        const Array<PrimExpr>& data_shape, const Array<PrimExpr>& target_shape) {
+  arith::Analyzer* analyzer = ctx->GetAnalyzer();
+
+  int data_ndim = data_shape.size();
+  int target_ndim = target_shape.size();
+
+  int data_ax = data_ndim - 1, target_ax = target_ndim - 1;
```

Comment:
Usually we define one variable per line to make it clearer.
```suggestion
  int data_ax = data_ndim - 1;
  int target_ax = target_ndim - 1;
```